hydromt_fiat/api/exposure_vm.py

```diff
@@ -131,6 +174,20 @@ def set_asset_locations_source(
             # make backend calls to create translation file with fiat_key_maps
             print(catalog_entry)
         # write to data catalog
+        elif source == "OSM":
+            # download OSM data
+            self.exposure_buildings_model = ExposureBuildingsSettings(
+                asset_locations=source,
+                occupancy_type=source,
+                max_potential_damage=max_potential_damage,
+                ground_floor_height=ground_floor_height,
+                unit=Units.meters.value,  # TODO: make flexible
+                extraction_method=ExtractionMethod.centroid.value,
+                damage_types=["structure", "content"],
+                damage_unit = "$",
```

Comment:
Is this dollars or Euros in the end?

Reply:
I cannot find a conversion form Eur to USD in the code and also the damage values are based on the attache d file which is in Eur/m-2
[copy_of_global_flood_depth-damage_functions__30102017.xlsx](https://github.com/Deltares/hydromt_fiat/files/15198781/copy_of_global_flood_depth-damage_functions__30102017.xlsx)